src/scripts/InstanceScripts/Wotlk/Raid_IceCrownCitadel.cpp

```diff
@@ -1216,6 +1178,71 @@ class BoneSlice : public SpellScript
     uint32_t targetCount;
 };
 
+//////////////////////////////////////////////////////////////////////////////////////////
+// Boss: Lady Deathwhisper
+//////////////////////////////////////////////////////////////////////////////////////////
+
+///////////////////////////////////////////////////////
+// Boss: Valithria Dreamwalker
+// ...
+//
+//
+//
+//
+
+///////////////////////////////////////////////////////
```

Comment:
//////////////////////////////////////////////////////////////////////////////////////////
// Boss: Gunship Battle Alliance

Reply:
You dont like it that way?

Reply:
stick to code style =) (although this is not the main thing in such a volume of changes)
https://ascemu.github.io/Wiki/docs/coding-standards/ Comments😉